metrics-server/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Build stage
+FROM quay.io/cybozu/golang:1.13-bionic AS build
+
+ARG METRICS_SERVER_VERSION=0.3.6
+ARG CGO_ENABLED=0
+WORKDIR /go/src/github.com/kubernetes-incubator
```

Comment:
In the task of `make all`, `GO111MODULE` is `on`, so you can probably build the binary at the default workspace.
This WORKDIR is not needed.

https://github.com/kubernetes-sigs/metrics-server/blob/master/Makefile#L47

Reply:
That's true for `master`, however its contents haven't been officially released yet. The latest release (`0.3.6`) didn't yet use go modules.

Reply:
Sorry !!! 🙇